src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -1917,7 +1917,7 @@ private void ReplaceProjectionSpans(ReadOnlyCollection<SnapshotSpan> oldProjecti
 
         private static ITrackingSpan CreateTrackingSpan(SnapshotSpan snapshotSpan)
         {
-            return new CustomTrackingSpan(snapshotSpan.Snapshot, snapshotSpan.Span, PointTrackingMode.Negative, PointTrackingMode.Positive);
+            return new CustomTrackingSpan(snapshotSpan.Snapshot, snapshotSpan.Span, PointTrackingMode.Negative, PointTrackingMode.Negative);
```

Comment:
Yes, this is the only change in behavior. We use { Negative, Positive } elsewhere, when creating a tracking span for the leading edge of the input buffer.